src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedObject = {};
+
+  for (const key in robot) {
+    if (inversedObject[robot[key]]) {
+      return null;
+    }
+    inversedObject[robot[key]] = key;
```

Comment:
you repeat robot[key] 2 times here, better create variable with this value

Reply:
done